src/util/logs.rs

```diff
@@ -0,0 +1,64 @@
+use crate::subscriptions;
+use colored::Colorize;
+
+pub fn format_attr_log(mut log: subscriptions::deployment_logs::LogFields) {
+    if !log.attributes.is_empty() {
+        let mut timestamp: Option<String> = None;
+        let mut level: Option<String> = None;
+        let message = log.message;
+        let mut others = Vec::new();
+        // for some reason, not all have "" around the value
+        for attr in &mut log.attributes {
+            if !attr.value.starts_with('"') {
+                attr.value.insert(0, '"');
+            };
+            if !attr.value.ends_with('"') {
+                attr.value.push('"');
+            }
+        }
+        // get attributes using a match
+        for attr in &log.attributes {
+            match attr.key.to_lowercase().as_str() {
+                "timestamp" | "ts" | "time" => {
+                    timestamp = Some(attr.value.clone().replace('"', ""))
+                }
+                "level" | "lvl" => level = Some(attr.value.clone()),
+                _ => others.push(format!(
+                    "{}{}{}",
+                    attr.key.clone().bright_cyan(),
+                    "=",
+                    attr.value
+                        .clone()
+                        .replace('"', "\"".dimmed().to_string().as_str())
+                )),
+            }
+        }
+        // get the level and colour it
+        let level = level.map(|level| {
+            // make it uppercase so we dont have to make another variable
+            // for some reason, .uppercase() removes formatting
+            let level = level.replace('"', "").to_uppercase();
+            match level.to_lowercase().as_str() {
+                "info" => level.blue(),
+                "error" => level.red(),
+                "warn" => level.yellow(),
+                "debug" => level.magenta(),
+                _ => level.normal(),
+            }
+            .bold()
+        });
+        println!(
+            "{}={} {}={} {}={}{}{5} {}",
+            "timestamp".bright_cyan(),
+            timestamp.unwrap_or_default().purple(),
+            "level".bright_cyan(),
+            level.unwrap_or_default(),
+            "msg".bright_cyan(),
+            "\"".dimmed(),
+            message,
+            others.join(" ")
+        );
```

Comment:
Thoughts on cleaning up this formatting a bit? I have a few thoughts:

- We probably don't need the labels for timestamp and level, since they're obvious on their own
- Message could be between level and attrs, differentiated by color
- Can we try parsing/reformatting the time to be something more readable? Similar to the UI like `Nov 17 13:55:34.123` would be great
- Level colored to match the severity (debug=gray, info=cyan, warn=yellow, error=red) then have the attr names be magenta

A good format (imo) would be something like:

![CleanShot 2023-11-17 at 14 26 19@2x](https://github.com/railwayapp/cli/assets/587576/fc4b3b2a-38dd-46c4-9398-9529eb012569)